angular.json

```diff
@@ -124,6 +124,46 @@
           }
         }
       }
+    },
+    "onto-search": {
+      "projectType": "library",
+      "root": "projects/onto-search",
+      "sourceRoot": "projects/onto-search/src",
+      "prefix": "onto",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:ng-packagr",
+          "options": {
+            "tsConfig": "projects/onto-search/tsconfig.lib.json",
+            "project": "projects/onto-search/ng-package.json"
+          },
+          "configurations": {
+            "production": {
+              "tsConfig": "projects/onto-search/tsconfig.lib.prod.json"
+            }
+          }
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
+          "options": {
+            "main": "projects/onto-search/src/test.ts",
+            "tsConfig": "projects/onto-search/tsconfig.spec.json",
+            "karmaConfig": "projects/onto-search/karma.conf.js"
```

Comment:
We should consider removing the karma dependency some day as we don't plan on writing unit tests with it.